flow/project.py

```diff
@@ -3590,6 +3590,7 @@ class MyProject(FlowProject):
         parser_submit = subparsers.add_parser(
             'submit',
             parents=[base_parser],
+            conflict_handler='resolve',
```

Comment:
The default Slurm environment now defines a `--memory` option with no default value, and the Great Lakes environment defines a `--memory` option with a default of `4g`. To allow the Great Lakes option to override the default option, we needed to use `conflict_handler='resolve'`. I tested specifying and not specifying a `--memory` option for a default Slurm cluster and on Great Lakes, and it respected the default (or lack of default) properly.